enzyme/test/Enzyme/ReverseMode/remainder.ll

```diff
@@ -0,0 +1,38 @@
+; RUN: %opt < %s %loadEnzyme -enzyme -enzyme-preopt=false -O3 -S | FileCheck %s
```

Comment:
This and the above tests still don't seem updated?

Reply:
Have it locally. I didn't finish the other update yet and I generally try to push multiple commits simultaneously to not wait so long for CI.